### ISSUE TITLE
psort: questions about usage and empty csv with time slice

### ISSUE THREAD

**Issue by MarkDubs123** [log2timeline/plaso#1783]
Hi all im trying to sort my test.plaso file that I created I keep getting a dynamic requires an output file. I followed the guide for psort under wiki. 

$ psort.py -q --slice "2018-03-28 12:25:00" --slice_size 100 test.plaso
ERROR: Output format: dynamic requires an output file

usage: psort.py [-h] [-V] [--analysis PLUGIN_LIST] [--disable_zeromq]
                [--temporary_directory DIRECTORY] [--worker-memory-limit SIZE]
                [--logfile FILENAME] [-d] [-q] [--status_view TYPE]
                [--slice DATE] [--slice_size SLICE_SIZE] [--slicer]
                [--data PATH] [-a] [--language LANGUAGE] [-z TIMEZONE]
                [-o FORMAT] [-w OUTPUT_FILE] [--fields FIELDS]
                [--additional_fields ADDITIONAL_FIELDS]
                [--profilers PROFILERS_LIST] [--profiling_directory DIRECTORY]
                [--profiling_sample_rate SAMPLE_RATE]
                [STORAGE_FILE] [FILTER]

If you know how to fix this please show me the command that would correct this.

**Comment by joachimmetz** [comment]
Add `w output.log`. See: https://github.com/log2timeline/plaso/wiki/Using-psort

**Comment by MarkDubs123** [comment]
Any chance you can give me the command if I wanted to do it the following way in the previous command but make it csv.

**Comment by joachimmetz** [comment]
It is not clear to me what you are asking me. CSV is an output format. The "dynamic" output module creates CSV. See: https://github.com/log2timeline/plaso/blob/master/test_data/end_to_end/dynamic.log

Can you explain to me in plain English what it is you are trying to accomplish and where you think we can help you?

**Comment by MarkDubs123** [comment]
psort.py -o l2tcsv -w test.csv -z UTC test.plaso -q --slice '2018-03-28 12:25:00' --slice_size 100
 
I run that but it only generates a empty csv file why is that?

**Comment by joachimmetz** [comment]
that is a different issue then you initially reported can you provide me with the follow additional data so I might be able to pinpoint the issue:

**Source data:**

Please provide the source data you used when you experienced the problem.
For publicly available data please provide an URL or path of the source data.

**Plaso version:**

For example 20171231

**Operating system Plaso is running on:**

Not the operating system of the image/files you're trying to analyze.

**Installation method:**

For example:
* installed from [GiFT PPA][https://launchpad.net/~gift] stable track
* installed from [GiFT COPR][https://copr.fedorainfracloud.org/coprs/g/gift/] stable track
* installed from [l2tbinaries][https://github.com/log2timeline/l2tbinaries] master branch
* built using [l2devtools][https://github.com/log2timeline/l2tdevtools]
* other, namely ...

If multiple methods were used please indicate.

**Debug output/tracebacks:**

Any relevant output.

**Comment by MarkDubs123** [comment]
source data - download psort.py file from gitbuh 
plaso version - most recent 
OS- MAC
method - downloaded the psort file saved it to desktop and thats it

**Comment by joachimmetz** [comment]
> plaso version - most recent

Can you be specific, most recent as in e88746aeadf521fb41837aea1ad68c0c7d95603f (HEAD) or release version 20180127 ?

> source data

where does test.plaso come from? how did you generate it? what was the input data for the log2timeline.py run that created it?

> method - downloaded the psort file saved it to desktop and thats it

plaso will not run successfully without dependencies, use one of the recommended installation methods, or we cannot help you. See: https://github.com/log2timeline/plaso/wiki/Users-Guide

**Comment by joachimmetz** [comment]
No response from original reporter considering questions answered
